docs/docs/sample-scripts/spo/copy-listitems-sharepointlist.md

```diff
@@ -0,0 +1,34 @@
+# Copy list items between SharePoint lists
+
+Author: [SekThang](https://github.com/SekThang), Inspired by [Ruud](https://lazyadmin.nl/it/copy-sharepoint-list-items-to-another-list-with-powershell-and-pnp/)
+
+The cli script helps you to copy list items from one list to another list.
+I have written script logics to migrate list items from one site collection to another site collection
+
+- Prerequisites: List and metadata should be created in the destination site collection site as it's in the source site collection
+- Please update columns as per your needs in the line number 24 and 26
+
+```powershell tab="PowerShell"
+$SourceSite = Read-Host -Prompt 'Source site Url'
+$DestinationSite = Read-Host -Prompt 'Destination site Url'
+$SourceList = Read-Host -Prompt 'Source list name'
+$DestinationList = Read-Host -Prompt 'Destination list name'
+
+$listItems = m365 spo listitem list --title $SourceList --webUrl $SourceSite --output json | ConvertFrom-Json
+Write-Host 'Total count in the source list is'-> -fore Green $listItems.Count
+$count = 0
+$reportLocation = Get-Location
+foreach($item in $listItems)
+{
+	  $count++
+		m365 spo listitem add --listTitle $DesitnationList --webUrl $DestinationSite --Title $item.Title --Firstname $item.Firstname --Lastname $item.Lastname
```

Comment:
Hi @SekarThangavel  - There is a typo in the Destination List Variable. I am getting an error hence. In the command, Destination list variable is mentioned `$DesitnationList` while it is supposed to be **`$DestinationList`**. Guess it should work after the change